finagle-netty4/src/main/scala/com/twitter/finagle/netty4/ssl/server/Netty4ServerSslHandler.scala

```diff
@@ -53,8 +53,11 @@ private[netty4] class Netty4ServerSslHandler(
     // create an `SslHandler`.
     new SslHandler(engine.self)
 
-  private[this] def addHandlerToPipeline(pipeline: ChannelPipeline, sslHandler: SslHandler): Unit =
+  private[this] def addHandlersToPipeline(pipeline: ChannelPipeline, sslHandler: SslHandler,
```

Comment:
we use a slightly different style here:

    private[this] def addHandlersToPipeline(
      pipeline: ChannelPipeline,
      sslHandler: SslHandler,
      sslConnectHandler: SslServerConnectHandler
    ): Unit = {